src/Alpha_complex/include/gudhi/Alpha_complex_3d.h

```diff
@@ -567,9 +573,32 @@ Weighted_alpha_complex_3d::Weighted_point_3 wp0(Weighted_alpha_complex_3d::Point
     return true;
   }
 
+  /** \brief get_point returns the point corresponding to the vertex given as parameter.
+   *
+   * @param[in] vertex Vertex handle of the point to retrieve.
+   * @return The point found.
+   * @exception std::out_of_range In case vertex is not found (cf. std::vector::at).
+   */
+  const Point_3& get_point(std::size_t vertex) {
```

Comment:
Not making this method `const` (with a mutable `cgal_vertex_iterator_vector`) advertises the fact that this is not thread-safe, ok.